hw/net/rocker/rocker_world.h

```diff
@@ -56,5 +59,6 @@ enum rocker_world_type world_type(World *world);
 const char *world_name(World *world);
 
 World *rocker_get_world(Rocker *r, enum rocker_world_type type);
+World *rocker_world_from_pport(int pport);
```

Comment:
not anymore. will remove.

On Sun, Apr 19, 2015 at 9:35 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/rocker/rocker_world.h
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28663678>:
>
> > @@ -56,5 +59,6 @@ enum rocker_world_type world_type(World *world);
> >  const char *world_name(World *world);
> >
> >  World *rocker_get_world(Rocker *r, enum rocker_world_type type);
> > +World *rocker_world_from_pport(int pport);
>
> Is this used anywhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28663678>.
>